scripts/Muon/GUI/Common/phase_table_widget/phase_table_presenter.py

```diff
@@ -144,37 +144,38 @@ def handle_calculation_success(self):
 
     def calculate_phase_table(self):
         parameters = self.create_parameters_for_cal_muon_phase_algorithm()
+        fitting_workspace_name = get_fitting_workspace_name(parameters['DetectorTable'])\
+            if self.view.output_fit_information else '__NotUsed'
```

Comment:
are we ok to use `_NotUsed`?

Reply:
Yes I think so as in this case we don't care about this output. It will be output to the ADS anyway but will be hidden